src/java.base/windows/classes/sun/nio/fs/WindowsPath.java

```diff
@@ -174,16 +174,25 @@ String getPathForPermissionCheck() {
     // use this path for Win32 calls
     // This method will prefix long paths with \\?\ or \\?\UNC as required.
     String getPathForWin32Calls() throws WindowsException {
+        return getPathForWin32Calls(false);
+    }
+
+    String getPathWithPrefixForWin32Calls() throws WindowsException {
+        return getPathForWin32Calls(true);
+    }
+
+    String getPathForWin32Calls(boolean alwaysAddPrefix)
+        throws WindowsException {
```

Comment:
Minor nit, can you move the "throws WindowsException {" to the previous line.

Reply:
Fixed.